timeline-chart/src/components/time-graph-row-element.ts

```diff
@@ -16,6 +16,7 @@ export class TimeGraphRowElement extends TimeGraphComponent {
     height: number;
     position: TimeGraphElementPosition;
     textWidth: number = 0;
+    MAX_HEIGHT: number = 19;
```

Comment:
Where do 19 and 14 below come from? Just arbitrary value to make it work? If so, maybe you could add a fixme to improve general height calculation. I seem to recall having heard about "rows" that would span like 2 rows or that would have custom sizes eventually? Those hard-coded values will have to be updated then.

Reply:
I set the maximum state height to "1" less than the row height of the timeline chart. Row height for timeline chart is 20. That was the reason for setting up MAX_HEIGHT to 19. I replaced the hard coded value "19" by "_row.height -1" and pushed the code again.